profiles/models.py

```diff
@@ -59,6 +59,8 @@ def get_notification_context(self):
 @reversion.register()
 class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
It is not advisable to allow both `null=True` and `blank=True` on `CharField` or its descendants, as it can cause confusions and incosistencies with some empty values stored as blanks and some as nulls. Unfortunately, this was missed in earlier stages of the profile's development and now we have some `CharField` where both are allowed 😅 , like in the case of `username` a couple of rows down. Would be super neat, if somebody could fix those at some point. 🙏

Reply:
Yep. I was kind of following the convention as I'm not even sure if the case where those fields are null or blank is really valid. But these should be fixed (at some point).

Reply:
Could you drop `null=True` from these? at least new fields would be already safe 🙂 we can fix the old fields later